### ISSUE TITLE
Suggestion: Checklist item colour change

### ISSUE THREAD

**Issue by wildcate** [HabitRPG/habitica#2414]
First of all, yay checklists! So nice to finally have them!
My only (small) issue with the lists is that they seem to become redder even faster than normal to-do items, and then glare at me menacingly even though I am working as fast as I can... 

Would it be possible to reset the shading of a checklist to some more friendly colour (like green or even blue) once a checkbox has been clicked? That would show that work has been done on this, and be some instant feedback since the big payoff will only occur once the checklist is finished. (Plus it would be so nice to have some friendlier-coloured spots in my way too long, way too red list of to-dos!)

**Comment by colegleason** [comment]
I believe the scoring on checklists causes the to get more red faster since each checklist item is a multiplier. We should look at not applying the multiplier when changing the score down, perhaps.

**Comment by deilann** [comment]
This be fixed.
